src/styles/font-override-classes/break-word/index.njk

```diff
@@ -0,0 +1,19 @@
+---
+title: Break word – Typography
+layout: layout-example.njk
+---
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-one-half">
+    <p class="govuk-body">
+      <strong>With class</strong><br>
+      Email me on <span class="govuk-!-text-break-word">arthur_phillip_dent.42@peoplepersonalitydivision.siriuscyberneticscorporation.corp</span>.
```

Comment:
Any ideas for a relevant or plausible example? Our cabinet office emails sure are close enough.

Reply:
I know these are always tricky – we basically want to avoid somehow encouraging bad practice in with our examples. In this case it's inappropriate email address formats.

How about for now we just make it conform to our convention for email addresses? I haven't found it but I'd infer it'd be:

```suggestion
      Email me on <span class="govuk-!-text-break-word">arthur.philip.dent.42@people-personality-division.sirius-cybernetics-corporation.corp</span>.
```

Reply:
That doesn't work for the purposes of the example, unfortunately, as browsers interpret hyphens as safe break points by default. The email no longer 'breaks out' if you include them.

Reply:
As you can tell, I dislike negative examples! I suppose we can just camel casing the domain then?

```suggestion
      Email me on <span class="govuk-!-text-break-word">arthur.phillip.dent.42@PeoplePersonalityDivision.SiriusCyberneticsCorporation.corp</span>.
```

But! I just thought up that we could create a fictitious council domain for the long Welsh town name. Wondering if you have any thoughts to add on this topic @seaemsi?

```suggestion
      Email me on <span class="govuk-!-text-break-word">firstname.lastname@llanfairpwllgwyngyllgogerychwyrndrobwllllantysiliogogogochuchaf.gov.uk</span>.
```

Reply:
I like the welsh town name example. Happy to go with that if there's no other issues with it!

Reply:
Nice! Would something like "contact@..." be long enough? Otherwise I can try to figure out the naming convention...

Reply:
Actually, it'd be good if the example implies it's a user-entered email address (rather than something someone put in a service), so let me write a suggestion.

Something like: We'll email the address you entered: <long long email>

Reply:
Okie doke, this is 86 characters like Arthur's original email. How's this work?

```suggestion
      We'll send an email to:<span class="govuk-!-text-break-word">communications@llanfairpwllgwyngyllgogerychwyrndrobwllllantysiliogogogochuchaf.gov.uk</span>.
```

Reply:
It works pretty well, I think! I've updated the PR to use that example.

Reply:
As an aside, having some manner of standardised example data might be nice. This isn't the only example where having that would be useful. We already have a lot of [dummy data in the Summary list component examples](https://design-system.service.gov.uk/components/summary-list/).

Reply:
Oh interesting, yeah that sounds like a fine 'small story' piece of work. 

Obvs we try to write placeholder content as appropriate as possible, but "demonstrating view-type components in a design system for government services" is probably focussed enough.